client/my-sites/checkout/checkout-thank-you/akismet-checkout-thank-you.tsx

```diff
@@ -15,12 +16,21 @@ interface AkismetCheckoutThankYouProps {
 const AkismetCheckoutThankYou: FunctionComponent< AkismetCheckoutThankYouProps > = ( {
 	productSlug,
 } ) => {
+	const dispatch = useDispatch();
 	const hasProduct = productSlug !== 'no_product';
```

Comment:
I believe this page is based on `jetpack-checkout-thank-you.tsx` - that contains similar code - but I don't really know when/how `productSlug` could be `no_product`. @CodeyGuyDylan, did you test if that's possible for Akismet?

Reply:
No, I did not test to see if this were possible with Akismet. I don't know what circumstances that it happens in Jetpack either though. I am guessing this is a safeguard in case somebody manually removes the productSlug from the URL?